src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -75,6 +74,7 @@ endif
 
 if (! -e $SCRDIR            ) mkdir -p $SCRDIR
 
+set NENS = `grep NUM_LDAS_ENSEMBLE   $HOMDIR/LDAS.rc | cut -d':' -f2`
```

Comment:
Should there be a colon after "grep   NUM_LDAS_ENSEMBLE"?  There is a colon after the grep for END_DATE and NUM_SGMT.

Reply:
It is not necessary. The grep gets the whole line and cut by ":".  But we can add it if we want the syntaxs to be consistent.